src/components/TodoHeder.tsx

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
+import React from 'react';
+import cn from 'classnames';
+
+type Props = {
+  activeButton: boolean,
+};
+
+export const TodoHeder: React.FC<Props> = ({ activeButton }) => {
+  return (
+    <header className="todoapp__header">
+
+      <button
```

Comment:
```suggestion
    <header className="todoapp__header">
      <button
```
Don't leave empty lines between parent and child elements